### DISCUSSION TITLE
Firefox issue with sharing screen

### DISCUSSION THREAD

**Discussion by bemyevent** [mozilla/hubs#2986] [General Questions & Discussion] [answered]
We have a problem with sharing screen with firefox browser,  it is impossible to share screen,  have you a solution for that ? 
Thx

**Comment by emclaren** [comment, marked as answer]
Hi thanks for reporting this. To troubleshoot & test it would be great to get a bit more information: 

1) What type of device are you using (Windows/Mac computer)?
2) What happens when you try to screen share (does the button do nothing at all? is there a 'loading' icon and it doesn't turn into a screen?)

Please let us know so we can see if we can replicate the issue. In the mean time, you might find that updating your browser could resolve this issue.
